### ISSUE TITLE
Employee (not supervisor) can see that another employee is requesting events.

### ISSUE THREAD

**Issue by KuchaLeni** [SoftwareCountry/arcadian-assistant#488]
Employee (not supervisor) should be able to see only processed vacations, approved dayoffs and sickleaves of another employee.

**Comment by sponkrashin** [comment]
Are you sure this logic is correct? I suppose that ordinal employee could see only own events.

**Comment by sponkrashin** [comment]
@santee?

**Comment by santee** [comment]
within one department why not? Title is shit, i give you that.

**Comment by sponkrashin** [comment]
Title became more confusing. Bug is that employee could see, or couldn't, or could but not the information that it should?

**Comment by sponkrashin** [comment]
Ok, let's summarize that employee can see approved, completed and processed events of employees of the same department.

**Comment by sponkrashin** [comment]
@santee, are you sure we really need to do this now?

**Comment by santee** [comment]
no, not necessary now. but that should work already, shouldn't it? I thought we have a set of permissions returned for it.

**Comment by sponkrashin** [comment]
We do have only permission to read all events of employee.

**Comment by santee** [comment]
postpone it then
